solidity/contracts/bridges/Biconomy/ILiquidityPoolManager.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
```

Comment:
We probably can remove the bridges folder, since it's not used in the phase2 final implementation AFAIK